BlockchainSdk/Blockchains/Solana/SolanaWalletManager.swift

```diff
@@ -306,3 +306,34 @@ private extension SolanaWalletManager {
         let space: UInt64?
     }
 }
+
+
+// MARK: - StakeKitTransactionSender
+
+extension SolanaWalletManager: StakeKitTransactionSender {
+    func sendStakeKit(transaction: StakeKitTransaction, signer: any TransactionSigner) -> AnyPublisher<TransactionSendResult, SendTxError> {
+        let helper = SolanaStakeKitTransactionHelper()
+
+        return Just(helper.prepareForSign(transaction.unsignedData))
+            .withWeakCaptureOf(self)
+            .flatMap { manager, txToSign in
+                signer.sign(hash: txToSign, walletPublicKey: manager.wallet.publicKey)
+            }
```

Comment:
```suggestion
     let txToSign  = helper.prepareForSign(transaction.unsignedData)
     signer.sign(hash: txToSign, walletPublicKey: manager.wallet.publicKey)
```

можно без Just()
а после `signer.sign` уже можно через `map/flatMap` делать

Reply:
А я на такой подход переходил в EthereumWalletManager, чтобы был реактивный контекст и код выполнялся только при наличии потребителя.